tests/e2e/ctl_v3_lock_test.go

```diff
@@ -69,7 +69,12 @@ func testLock(cx ctlCtx) {
 	if err != nil {
 		cx.t.Fatal(err)
 	}
-	defer blockAcquire.Stop()
+	defer func(blockAcquire *expect.ExpectProcess) {
```

Comment:
Should this be in defer? Would putting it after select that reads from `ch` enough?

Reply:
true, it makes more sense to not defer stop here

Reply:
hmm, seems like there was some reason after all:
> 2022-11-17T13:44:28.5305567Z     ctl_v3_lock_test.go:106: expected different lock name, got l1="a/186d8485d2349505\r\n", l2="Error: context canceled\r\n"